src/tests/system/data/test_passkey/readme.md

```diff
@@ -0,0 +1,56 @@
+Passkey support is tested with su, tests are running with
```

Comment:
Let's use the same markup language as the other readme,

 https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax 
 
 The content looks great!

Reply:
I already use the same syntax but if we want format, please let us know I can do it.

Reply:
Looking that formatted page, it looks kind of strage. 

https://github.com/SSSD/sssd/blob/f1f123c7c132eabc685309fad9e61566bdf3c4e0/src/tests/system/data/test_passkey/readme.md

Let's swap out  `commands` with code blocks, so  prefix and postfix with ```

```
commands
```

line 8, 13-17, 36, 38, 42-47, 53-56